### PR TITLE
Add phone provider

### PR DESCRIPTION
**Pull request by agoalofalife**
I add new provider : phone( + tests)
I think need update readme file.

### CODE REVIEW

File: phone.go [hunk 1]
```diff
@@ -0,0 +1,68 @@
+package faker
+
+import (
+	"fmt"
+	"github.com/agoalofalife/faker/support/slice"
```
Comment (bxcodec) [inline phone.go]: This package, can confuse the package version manager. Change into
`github.com/bxcodec/faker/support/slice`

Comment (bxcodec) [reply phone.go]: wait. I think I will replaced it later. I'm afraid changing this will cause an error on travis

Comment (agoalofalife) [reply phone.go]: Yes it was replaced before

Comment (agoalofalife) [reply phone.go]: I have added a new feature, its now not in general repository

Comment (bxcodec) [reply phone.go]: Anymore addition ??, if not, I will merge this, and repair the packages..

Comment (agoalofalife) [reply phone.go]: Yes, it is possible to correct only after the merge

**Pull request merged**